Makefile

```diff
@@ -154,8 +159,27 @@ KUSTOMIZE = $(PROJECT_DIR)/bin/kustomize
 kustomize:
 	@GOBIN=$(PROJECT_DIR)/bin go install sigs.k8s.io/kustomize/kustomize/v4@v4.5.7
 
+# The build via `go install` will fail with the following err:
+# > The go.mod file for the module providing named packages contains one or
+#   more replace directives. It must not contain directives that would cause
+#   it to be interpreted differently than if it were the main module.
+# However, we can ignore the above error since it is only necessary to download the manifests.
+.PHONY: scheduler-plugins
+scheduler-plugins:
+	-@GOPATH=/tmp go install sigs.k8s.io/scheduler-plugins/cmd/controller@$(SCHEDULER_PLUGINS_VERSION)
```

Comment:
Is the `-` at the beginning of the line intentional?

Reply:
Yes, I added `-` to ignore the error. The command works similarly to the below:

```shell
@GOPATH=/tmp go install sigs.k8s.io/scheduler-plugins/cmd/controller@$(SCHEDULER_PLUGINS_VERSION) || :
```